openkfw/TruBudget
# 1836 cache global permissions api

### Checklist




### Description
* added caching of global permissions

### How to test

1. run the app, click through it and make sure nothing is broken




Closes #1836
